tests/unit/handlers/test_web.py

```diff
@@ -0,0 +1,130 @@
+import unittest
```

Comment:
This is not mandatory, but for the sake of keeping the code clean, I think it might be a good idea to move this test data to a new (config) file and move the tests that have been written for the helpers to a new class, maybe `TestURLCrawlHelpers`?